wordpress/wordpress-ha-aurora.yaml

```diff
@@ -121,6 +125,10 @@ Parameters:
     Type: Number
     Default: 14
     AllowedValues: [1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, 3653]
+  EFSProvisionedThroughputInMibps:
+    Description: 'The provisioned throughput for the Elastic File System (EFS) in Mibps. Default is 0.0 which enables the bursting mode and disables provisioned throughput.'
+    Type: Number
+    Default: 0.0
```

Comment:
why `0.0` and not `0`? `MinValue` is `0` ?

Reply:
Good question! :)